### DISCUSSION TITLE
Can't play any video, freetube is blocked: corrupted cache?

### DISCUSSION THREAD

**Discussion by daks** [FreeTubeApp/FreeTube#6350] [Q&A]
Since a few days I can't play any video on freetube. Once a click on one, the UI gets blocked, I can't click anywhere, I just still can scroll the current page.
I must kill the app to close it.

On terminal I see those messages
```
Error: ENOENT: no such file or directory, open '/home/eric/.config/FreeTube/player_cache/3bb1f723'
    at async open (node:internal/fs/promises:639:25)
    at async Object.readFile (node:internal/fs/promises:1242:14)
    at async /opt/FreeTube/resources/app.asar/dist/main.js:2:105760
    at async WebContents.<anonymous> (node:electron/js2c/browser_init:2:86160) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/home/eric/.config/FreeTube/player_cache/3bb1f723'
}
```
I removed all files in `.config/FreeTube/player_cache` to try to solve the problem. Without success.
How can I solve this problem? Is there a way to "soft reset" my config without losing parameters, subscriptions, bookmarks, etc?

**Comment by absidue** [comment]
This sounds like a duplicate of #6328

**Comment by daks** [comment]
@absidue in the issue you linked I see no one talking about the `player_cache` file problem I see. Moreover for me all videos are affected not only musical ones.
I'm not sure it's the same bug.

**Reply by absidue** [reply]
The reason nobody is talking about is that it is not a problem, all that says is that it didn't find the deciphering code in the cache and will fetch it from YouTube. Maybe we should remove logging that message so that people stop thinking it is relevant to anything. If you don't believe me you can check the code:
https://github.com/FreeTubeApp/FreeTube/blob/development/src/main/index.js#L1028-L1093

**Reply by daks** [reply]
After installing a nightly build my problem is gone.

> The reason nobody is talking about is that it is not a problem, all that says is that it didn't find the deciphering code in the cache and will fetch it from YouTube. Maybe we should remove logging that message so that people stop thinking it is relevant to anything. If you don't believe me you can check the code: https://github.com/FreeTubeApp/FreeTube/blob/development/src/main/index.js#L1028-L1093

Maybe this message could be visible only in DEBUG mode? Or more explicit like  'ERR_CACHE_MISS' or something equivalent

Anyway I can now go back to freetube, thanks
